st2common/st2common/services/datastore.py

```diff
@@ -79,7 +85,8 @@ def get_value(self, name, local=True):
         self._logger.audit('Retrieving value from the datastore (name=%s)', name)
 
         try:
-            kvp = client.keys.get_by_id(id=name)
+            params = {'decrypt': str(decrypt).lower(), 'scope': scope}
+            kvp = client.keys.get_by_id(id=name, params=params)
```

Comment:
Then for now, can you throw an exception if someone passes scope=USER_SCOPE to the API call?